components/gfx/platform/windows/font_template.rs

```diff
@@ -81,6 +81,9 @@ impl FontTemplateData {
 
     /// Get a [`FontFace`] for this font if it is a local font or return `None` if it's a
     /// web font.
+    ///
+    /// [`FontFace`]: https://developer.mozilla.org/en-US/docs/Web/API/FontFace
+    ///
```

Comment:
This isn't the `FontFace` that's referred to here. You can just use `dwrote::FontFace`, I think.

Reply:
Actually, the issue here is that the method is returning `Font`, but I had written `FontFace` when I added this documentation because it changed while refactoring.

Reply:
Got it. Thanks for letting me know.